src/java.base/share/classes/sun/launcher/LauncherHelper.java

```diff
@@ -170,7 +170,7 @@ static void showSettings(boolean printToStderr, String optionFlag,
                 printProperties();
                 break;
             case "locale":
-                printLocale();
+                printLocale(false);
```

Comment:
Adding the flag seems to make the coding more complex.
I'd remove the printing of all locales from the printLocale method (and maybe rename it to printLocalSummary).
And add the printing of the help test for the the verbose here also.